.travis.yml

```diff
@@ -1,13 +1,69 @@
 
 jobs:
   include:
-    - os: linux
-      language: java
+    - name: Build - Linux
+      os: linux
       dist: trusty
       sudo: true
+      language: java
       jdk: openjdk11
-    - os: windows
+      env: DEPLOY=true
+    - name: Build - Windows
+      os: windows
       language: shell
+    - name: ITests - PostgreSQL
+      os: linux
+      dist: focal
+      sudo: true
+      language: java
+      jdk: openjdk11
+      addons:
+        postgresql: '12'
+        apt:
+          packages:
+            - postgresql-12
+            - postgresql-client-12
+      env:
+        - DB=postgresql
+        - DB_PORT=5432
+        - ITEST=true
+      before_script:
+        - psql -c 'CREATE DATABASE fcrepo;' -U postgres
+    - name: ITests - MariaDB
+      os: linux
+      dist: trusty
+      sudo: true
+      language: java
+      jdk: openjdk11
+      addons:
+        mariadb: "10.4"
+      env:
+        - DB=mariadb
+        - DB_PORT=3306
+        - ITEST=true
+      before_script:
+        - mysql -e 'CREATE DATABASE IF NOT EXISTS fcrepo;'
+    - name: ITests - MySQL
+      os: linux
+      dist: xenial
+      sudo: true
+      language: java
+      jdk: openjdk11
+      env:
+        - DB=mysql
+        - DB_PORT=3306
+        - ITEST=true
+      services:
+        - mysql
+      before_script:
+        - wget https://repo.mysql.com//mysql-apt-config_0.8.10-1_all.deb
```

Comment:
doesn't seem to make a difference here, but might be a good idea to remove the double slash

Reply:
fixed